scripts/health.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+to_status() {
+  if [[ $1 == 0 ]]; then
+    printf "✅"
+    return
+  fi
+  printf "❌"
+}
+
+check_go_mod_tidy() {
+  if [[ ! -f go.mod || ! -f go.sum ]]; then
+    echo 1
+    return
+  fi
+  cp go.mod go.mod.orig
+  cp go.sum go.sum.orig
+  go mod tidy &> /dev/null
+  diff go.mod go.mod.orig > /dev/null
+  if [[ $? != 0 ]]; then
+    echo 1
+    return
+  fi
+  diff go.sum go.sum.orig > /dev/null
+  echo $?
```

Comment:
why echo? We could just do what we do for the other commands (capture the return status)